java/lang/xss_servlet_parameter.yml

```diff
@@ -0,0 +1,57 @@
+patterns:
+  - pattern: |
+      $<REQUEST>.$<METHOD>();
+    filters:
+      - variable: REQUEST
+        values:
+          - req
+          - request
```

Comment:
Here I am following our approach in other rules. I don't know if there's any way to limit this to e.g. `ServletRequest` and `HttpServletRequest` objects (this would be more precise).